osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerReadyButton.cs

```diff
@@ -83,6 +96,16 @@ private void updateButtonText()
                 else
                     countdownRemaining = countdown.TimeRemaining - timeElapsed;
 
+                if (countdownRemaining.Seconds <= 10 && (lastTickPlayed == null || lastTickPlayed != countdownRemaining.Seconds))
+                {
+                    countdownTickSample?.Play();
+
+                    if (countdownRemaining.Seconds <= 3)
+                        countdownTickFinalSample?.Play();
```

Comment:
This sample plays on the counts of 3, 2, 1 and 0, and I was half expecting the 0 count sample to be somehow different, e.g. higher in pitch, to signal that it's the end of the countdown, but it's not so, it plays exactly the same which feels... off? Maybe I've been conditioned to expect this by other games but yeah.

Reply:
I was also expecting a pitch shift, as it gets close to zero. As I mentioned on the resource PR I'm not a huge fan of the warning-bell style of this sound right now.

Reply:
Some feedback on the sounds:

![CleanShot 2022-04-04 at 05 19 33](https://user-images.githubusercontent.com/191335/161478736-f61825df-3357-4d5c-997a-f35b6d109e38.png)